tests/test_processors.py

```diff
@@ -253,7 +254,7 @@ def build_event(headers):
         return flag_robots(event)
 
     assert build_event(request_headers["user"])["is_robot"] is False
-    assert build_event(request_headers["machine"])["is_robot"] is False
+    assert build_event(request_headers["machine"])["is_robot"] is True
```

Comment:
Can you comment this line and add a link to https://github.com/inveniosoftware/counter-robots/issues/15 instead of changing to True?